_episodes_rmd/03-r-data-analysis.Rmd

```diff
@@ -358,33 +414,96 @@ gapminder_data %>%
+Notice here that we tell `pivot_wider()` which columns to pull the names we wish our new columns to be named from the year variable, and the values to populate those columns from the `smoke_pct` variable. (Again, neither of which have to be in quotes in the code when there are no special characters or spaces - certainly an incentive not to use special characters or spaces!) We see that the resulting table has new columns by year, and the values populate it with our remaining variables dictating the rows.
+
+## Plotting wide data
 
-Before we move on to more data cleaning, let's create the final gapminder dataframe we will be working with for the rest of the lesson! 
+Let's make a plot with our wide data comparing percent of smokers in 1990 to percent of smokers in 2010 to see how it has changed for each country.
 
-> ## Final Americas 2007 gapminder dataset
-> Read in the `gapminder_data.csv` file, filter out the year 2007 and the continent "Americas." Then drop the `year` and `continent` columns from the dataframe. Then save the new dataframe into a variable called `gapminder_data_2007`. 
+```{r}
+smoking %>%
+  group_by(country, continent, year) %>% 
+  summarize(smoke_pct = mean(smoke_pct)) %>%
+  select(country, continent, year, smoke_pct) %>%
+  pivot_wider(names_from = year, values_from = smoke_pct) %>% 
+  ggplot(aes(x = 1990, y = 2010)) +
+  geom_point()
+```
+
+Hmm that's not what we want. `ggplot` just plotted the numbers 1952 and 2007 instead of the data from the years. That's because it evaluates those as numbers instead of column names. To fix this, we can add a prefix to the years in `pivot_wider()`:
+
+```{r}
+smoking %>%
+  group_by(country, continent, year) %>% 
+  summarize(smoke_pct = mean(smoke_pct)) %>%
+  select(country, continent, year, smoke_pct) %>%
+  pivot_wider(names_from = year, values_from = smoke_pct, names_prefix = 'y') %>% 
+  ggplot(aes(x = y1990, y = y2010)) +
+  geom_point()
+```
+
+That's better, but the different ranges on the x and y axis make it hard to compare the points. 
+Let's fix that by adding a line at y=x. 
+
+```{r}
+smoking %>%
+  group_by(country, continent, year) %>% 
+  summarize(smoke_pct = mean(smoke_pct)) %>%
+  select(country, continent, year, smoke_pct) %>%
+  pivot_wider(names_from = year, values_from = smoke_pct, names_prefix = 'y') %>% 
+  ggplot(aes(x = y1990, y = y2010)) +
+  geom_point() +
+  geom_abline(intercept = 0, slope = 1)
+```
+
+It seems like in most countries the percent of smokers has decrased from 1990 to 2010, but there are some countries where smoking has increased. Let's figure out which those are!
```

Comment:
```suggestion
It seems like in most countries the percent of smokers has decreased from 1990 to 2010, since most of the points fall below the line y = x. However, there are some countries where smoking has increased (i.e. the points are above the line y = x). Let's figure out which those are!
```